Simplenote/src/main/res/values/strings.xml

```diff
@@ -271,6 +271,11 @@
     <string name="description_up">Up</string>
     <string name="description_warning">Warning</string>
 
+    <string name="dialog_browser_exception_button_copy_url">Copy URL</string>
+    <string name="dialog_browser_exception_message">An app could not be opened with that link.</string>
```

Comment:
np: The Material Guidelines recommend [skipping unnecessary punctuation ](https://material.io/design/communication/writing.html#principles)such as periods at the end of a single sentence.

Reply:
Thanks for pointing that out.  That makes sense for toasts, but I'm surprised that's a recommendation for dialogs to be honest.  I'm not entirely sure that's the greatest message for the dialog anyway.  That's why I requested @SylvesterWilmott's review.